app/controllers/concerns/rate_limit_concern.rb

```diff
@@ -45,9 +45,6 @@ def rate_limit_redirect!(rate_limit_type)
 
   def track_rate_limited_event(rate_limit_type)
     analytics_args = { limiter_type: rate_limit_type }
-    limiter_context = rate_limit_type == :proof_ssn ? 'multi-session' : 'single-session'
-
-    irs_attempts_api_tracker.idv_verification_rate_limited(limiter_context: limiter_context)
     analytics.rate_limit_reached(**analytics_args)
```

Comment:
since we're here, maybe we just inline the args for this analytics method?

```suggestion
    analytics.rate_limit_reached(limiter_type: rate_limit_type)
```

Reply:
> since we're here, maybe we just inline the args for this analytics method?

Yeah, makes sense. At that point, I don't think we really need the extra method at all, so I just inlined it all in 5f940f3.